Bottom: 6631e84b8d68a2bcaf3f10aa34c81da46c1474ea
Top:    4a8cee00cdf2bce495b951f48f9c61d9dba70e59
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:54:58 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
